openstack/intel-worker/playbooks/roles/virtual_machine_create/tasks/ignition.yml

```diff
@@ -14,8 +14,8 @@
 
 - name: Set the variables (To be used in Customization Script)
   ansible.builtin.set_fact:
-    worker_hostname_encoded: "{{ worker_hostname | b64encode }}"
-    dns_none_encoded: "{{ dns_none_conf | b64encode }}"
+    worker_hostname_encoded: "{{ virtual_machine_create_worker_hostname | b64encode }}"
```

Comment:
```suggestion
    worker_hostname_encoded: "{{ worker_hostname | b64encode }}"
```

Reply:
please use worker_hostname. it's more readable

Reply:
Ansible lint expects to have a role name as a prefix for the variable names .